src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1302 @@
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 013@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_allmstatus(void)
+{
+    gTestID = 13;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int32_t select = 0;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    bool allmstatus = false;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    UT_LOG_INFO("\n**********Please select port***********\n");
+
+    readInput(&select);
+    if(select <= 0 || select > (numInputPorts -1))
+    {
+      UT_LOG_ERROR("\n invalid port selected\n");
+    }
+    port = (select - 1);
+
+    ret = dsGetAllmStatus(port , &allmstatus);
+    UT_LOG_INFO("Result : Passed dsGetEdidVersion port type: %s, portnumber: %s\t"
+                         "retrieved allm status: %s",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port), port,
+                 UT_Control_GetMapString(bool_mapTable, allmstatus));
+    assert(ret == dsERR_NONE);
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get supported game features list.
+*
+* This test function is to get supported game features list of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 014@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_gamefeatureslist(void)
+{
+    gTestID = 14;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsSupportedGameFeatureList_t features_list;
+    char *token = NULL;
+
+    ret = dsGetSupportedGameFeaturesList( &features_list);
+    UT_LOG_INFO("Result : dsGetSupportedGameFeaturesList retrieved features_list count: %d",
+                 features_list.gameFeatureCount);
+    assert(ret == dsERR_NONE);
+          
+    token = strtok(features_list.gameFeatureList , ",");
+    while(token != NULL)
+    {
+       UT_LOG_INFO("Result: Supported featureslist: %s", token);
+       token = strtok(NULL , ",");
+    }
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get AV latency.
+*
+* This test function is to get AV latency of HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 015@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_avlatency(void)
+{
+    gTestID = 15;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int audio_latency = 0, video_latency = 0;
+   
+    ret = dsGetAVLatency(&audio_latency , &video_latency);
```

Comment:
No pritns added before and need to mention IN and OUT

Reply:
OUT param should be empty as below , as we are calling before the function. 

 UT_LOG_INFO("Calling dsGetAVLatency OUT:audio_latency:[%d ms], OUT:video_latency:[]",
                         audio_latency );

Reply:
done